manage/sawtooth_manage/subproc.py

```diff
@@ -106,3 +118,19 @@ def is_running(self, node_name):
 
     def create_genesis_block(self, node_args):
         pass
+
+    def _rm_wildcard(self, path, pattern):
+        for each in os.listdir(path):
+            if pattern.search(each):
+                name = os.path.join(path, each)
+                try:
+                    os.remove(name)
+                except:
+                    traceback.print_exc(file=sys.stderr)
+                    sys.exit(1)
+
+    def _calc_peers(self, node_num, base_gossip_port):
```

Comment:
This could just be done as a list comprehension.  At least, the name should be something more descriptive.